Resources/Prototypes/_Impstation/Entities/Objects/Weapons/Guns/Special/basilisk.yml

```
@@ -0,0 +1,81 @@
+- type: entity
+  name: Basilisk
+  parent: [BaseItem, BaseGunWieldable]
+  id: WeaponSpecialBasilisk
+  description: This odd weapon fires off bursts of pure radiation, using echion as fuel.
+  components:
+  - type: Sprite
+    sprite: _Impstation/Objects/Weapons/Guns/Special/Basilisk.rsi
```

capitalization matters
```suggestion
    sprite: _Impstation/Objects/Weapons/Guns/Special/basilisk.rsi
```

---

I thought I fucking fixed that :elp: